content/en/database_monitoring/guide/sql_alwayson.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Exploring SQL Server AlwaysOn Availability Groups
+kind: documentation
+further_reading:
+- link: "/database_monitoring/"
+  tag: "Documentation"
+  text: "Database Monitoring"
+- link: "/database_monitoring/setup_sql_server/"
+  tag: "Documentation"
+  text: "Setting Up SQL Server"
+- link: "/database_monitoring/troubleshooting/"
+  tag: "Documentation"
+  text: "Troubleshooting Database Monitoring"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+The Database Monitoring AlwaysOn Clusters view enables you to detect data synchronization issues, understand availability group behavior, and identify cluster bottlenecks in SQL Server availability groups.
+
+To access the AlwaysOn Clusters view, navigate to the **APM** > **Database Monitoring** > **Databases** tab and select **AlwaysOn Clusters**.
+
+## Determine the health of your nodes
+
+Use the AlwaysOn Clusters view to evaluate the health of your SQL Server availability groups. When selected, the page shows a color-coded visualization based on the current status of the primary (P) and secondary (S) nodes in each availability group.
+
+To identify groups that are experiencing issues, use the status filters to show groups with nodes that are Reverting, Not Synchronizing, and so forth. You can also use the timeseries graphs to spot unusual performance activity in your clusters based on log, redo, and secondary lag time metrics.
```

Comment:
I can't find any guidance on this in our style guides, but should these UI terms (Reverting, Not Synchronizing) be italicized/bold? I think I'd usually bold them, but it's up to you.